noetic/distribution.yaml

```diff
@@ -898,6 +898,28 @@ repositories:
       url: https://github.com/fkie/message_filters.git
       version: master
     status: maintained
+  fkie_multimaster:
```

Comment:
Did you mean to call the repositoriy `multimaster_fkie`? It looks like that's what it was called in the past few ROS distros and how it's named on github: https://github.com/fkie/multimaster_fkie

* https://github.com/ros/rosdistro/pull/21058 - Melodic
* https://github.com/ros/rosdistro/pull/21057 - Kinetic
* https://github.com/ros/rosdistro/pull/13730 - Jade
* https://github.com/ros/rosdistro/pull/21056 - Indigo

Reply:
We renamed all packages inside this repository and redesigned a lot of functionality. So they are no fully compatible with previous versions. The packages have also dependencies (grpc) which are available since foxy/noetic.
For this reason, I also changed the repository name.

Reply:
For what it is worth, I find this super-confusing.  I think any casual user not intimately involved in the multimaster_fkie repository will have a hard time telling "multimaster_fkie" and "fkie_multimaster" apart.  For the sake of your users, I'd highly suggest calling this something like `multimaster_fkie2`, or `multimaster_fkie_ng`, or something like that.

That being said, if you want to continue with the current name, you can.  I'll leave this to @sloretz to finish reviewing and merging.